client/components/emojify/index.jsx

```diff
@@ -31,11 +31,15 @@ export default class Emojify extends PureComponent {
 		this.parseEmoji();
 	}
 
+	setRef( component ) {
```

Comment:
By using the fat arrow syntax here, 
```
setRef = ( component ) => {
```
we should be able to eliminate the `this.setRef.bind( this )` in the constructor.

Reply:
I know! But I'm trying to refrain from using `stage2` features.